### ISSUE TITLE
Hello, I use Java to load my training model, the program is stuck in “module.forward（）” this step is not gone, how to do?

### ISSUE THREAD

**Issue by niushaoda** [pytorch/android-demo-app#24]

**Comment by nmfisher** [comment]
I've got the same problem with both this repository and a separate application with a custom model - the application freezes at the call to module.forward(). 

There's nothing notable in logcat, so I've got no idea where to even begin debugging (save recompiling with debug enabled?).

**Comment by niushaoda** [comment]
Sometimes I can run it, sometimes I can't, I don't know if it's the android emulator or the model.pt.

**Comment by niushaoda** [comment]
Sometimes I can run it through here, sometimes I can't. I wonder if it is the reason of android emulator or "model.pt"

**Comment by lilinxiong** [comment]
I had the same problem. But I repair, in **HelloWorld** "app/src/main/asstes/image.jpg" size 400 * 400. But **PyTorchDemoApp** use input image size is 224 * 224. Only need to edit **HelloWorld** "app/src/main/asstes/image.jpg" size 224 * 224, run successful!

**Comment by IvanKobzarev** [comment]
Sorry for my late reply.

Torchvision pretrained models has input requirements to be at least 224.

> All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224.
https://pytorch.org/docs/stable/torchvision/models.html

So size 400x400 should ok for them.

There was deadlock issue in mobile build, that might be the reason of this hanging. It was fixed last week.
issue: https://github.com/pytorch/pytorch/issues/29821
commit: https://github.com/pytorch/pytorch/commit/b730d04ed246d5e9370b1c791432cdf242430dd9

As it was intermittent issue with race conditions - changing the size could help to avoid it.

Please check the latest nightly builds if it is still a problem or not.
To try nightly builds:

```
repositories {
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}

dependencies {
    ...
    implementation 'org.pytorch:pytorch_android:1.4.0-SNAPSHOT'
    implementation 'org.pytorch:pytorch_android_torchvision:1.4.0-SNAPSHOT'
    ...
}
```
